services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/BoosterNotificationStructureProvider.java

```diff
@@ -0,0 +1,80 @@
+package app.coronawarn.server.services.distribution.assembly.component;
+
+
+import app.coronawarn.server.services.distribution.assembly.structure.Writable;
+import app.coronawarn.server.services.distribution.assembly.structure.WritableOnDisk;
+import app.coronawarn.server.services.distribution.assembly.structure.archive.ArchiveOnDisk;
+import app.coronawarn.server.services.distribution.assembly.structure.archive.decorator.signing.DistributionArchiveSigningDecorator;
+import app.coronawarn.server.services.distribution.assembly.structure.file.FileOnDisk;
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
+import app.coronawarn.server.services.distribution.dgc.BusinessRule.RuleType;
+import app.coronawarn.server.services.distribution.dgc.DigitalGreenCertificateToCborMapping;
+import app.coronawarn.server.services.distribution.dgc.DigitalGreenCertificateToProtobufMapping;
+import app.coronawarn.server.services.distribution.dgc.dsc.DigitalSigningCertificatesToProtobufMapping;
+import app.coronawarn.server.services.distribution.dgc.exception.DigitalCovidCertificateException;
+import app.coronawarn.server.services.distribution.dgc.exception.FetchBusinessRulesException;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * Reads Booster notification rules from the respective files in the class path or from dcc client and build a {@link
+ * BoosterNotificationStructureProvider} with them.
+ */
+
+@Component
+public class BoosterNotificationStructureProvider {
+
+  private static final Logger logger = LoggerFactory.getLogger(BoosterNotificationStructureProvider.class);
+
+  public static final String EXPORT_BIN = "export.bin";
```

Comment:
Could we have a semantic name, such as EXPORT_BINARY_FILENAME or similar? As is, if the string changes, this needs to change as well...